### ISSUE TITLE
Inconsistent metadata handling for Image & HTML fields

### ISSUE THREAD

**Issue by graybeal** [metadatacenter/cedar-project#399]
The metadata for the Image and HTML fields is not handled consistently, and not as I'd expect.

For Images, the field name is shown as bold text under the image, and if no field name is entered, "unknown" is shown as bold under the image. This formatting is not useful that I can tell (I'd want it centered and/or on top, and not to appear if it is empty).

The field name for HTML is not shown, which is the better option in my opinion. 

The description is not shown in either case, so why is it there?

And for youtube entries, it is clear that most of those fields are optional (which suggests they will not be shown if not entered). This is optimal, but as suggested above, I don't think field name should ever be shown (even if entered), because it is clearly useful for labeling this content as an annotation, but not useful to present visually (because the presentation style can not be modified, and its presentation may be entirely unwanted).

**Comment by willrett** [comment]
You are suggesting that we hide the image title  in the metadata editor.  Images often have a caption beneath them to give the image some context.

The field name and description are useful in the template editor.  The description tooltip is displayed  when you hover over the question icon when the field is not under edit.

**Comment by graybeal** [comment]
You are correct that field name and description can be useful in the template editor. I wasn't thinking of that use case. And good to know the description can still be visible as well.

I think some of that doesn't accurately capture my suggestion. Let me try again.

* If the title is empty, it should not be shown as "Untitled", but instead should not be shown. 
* If there is a title, display it in the metadata editor as one would expect the title of an image to be displayed, which I think is centered and/or on top. 

These are not critically important requests. I am trying to make the fields as useful as possible for presenting CEDAR, and I would use them for that a lot if we can get the behaviors to match that use case. I am not aware of anyone else using these fields much at all, but there may be other use cases that could be more valuable than mine.

**Comment by willrett** [comment]
I use use HTMP5 figure and caption to display the image and title.  It is unusual to see the caption above the image.

**Comment by graybeal** [comment]
Yeah, I see how we have a clash of purposes, or perhaps of models, between treating these as "metadata fields" (as they could be in CEDAR); treating them as "documentation fields" (e.g., an HTML page); and treating them as "presentation fields". The first two may be at cross purposes with the presentation context I was pushing (i.e., in presentation slides, titles are almost always on top). 

These fields won't used for either a formal documentation or HTML document, where captions are under any graphical content and there is text above and below.  And I'm not sure there's any precedent for them in metadata documents. So my preference is to make them useful for presentation, since I know of at least two users for that (me, and Michelle F).

I'll go ahead and sign this off, because we have a lot of other issues about these fields. I created #825 to collect the list and describe the current status; resolving that issue will likely provide a path to resolve this as well.
